students/tbrackney/Lesson8/circle_class.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+"""
+File Name: circle.py
+Author: Travis Brackney
+Class: Python 201 - Self paced online
+Date Created 5/16/2018
+Python Version: 3.6.4
+"""
+import math
+
+
+class Circle():
+    """Implements a class of circle objects"""
+    def __init__(self, the_radius):
+        self._r = the_radius
```

Comment:
I would use `self.radius` for the attribute name for readability. The leading underscore denotes a 'protected' attribute and I'm not sure that's what you want to indicate here.